content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -226,6 +259,8 @@ You must use `ACS-Engine` to deploy your cluster.
     $ wget https://raw.githubusercontent.com/Azure/acs-engine/master/examples/service-mesh/istio.json
```

Comment:
Consider making sentence less ambiguous:

1. Download the `acs-engine` `api model definition` that supports deploying Istio :

Also the `https://raw.githubusercontent.com/Azure/acs-engine/master/examples/service-mesh/istio.json` example references `1.9`. Should we update this to align all the versions in this example?

Reply:
Yes, it would be good to update to use 1.10.x